### ISSUE TITLE
KS scraper broken

### ISSUE THREAD

**Issue by jzohrab** [covidatlas/coronadatascraper#916]
`yarn start --location US/KS` fails with:

```
Begin scraper for iso2:US-KS, iso1:US
  🐢  Cache miss for scraper: /US/KS/index.js; url: https://public.tableau.com/views/COVID-19Data_15851817634470/CountyCounts.pdf?:showVizHome=no; filepath: coronadatascraper-cache/2020-4-30/ac48861604ff931fc999c34e7558251b.pdf
  🚦  Loading data for https://public.tableau.com/views/COVID-19Data_15851817634470/CountyCounts.pdf?:showVizHome=no from server
  ❌ Got error 404 trying to fetch https://public.tableau.com/views/COVID-19Data_15851817634470/CountyCounts.pdf?:showVizHome=no
  ❌ Error processing iso2:US-KS, iso1:US:  TypeError: Cannot read property 'filter' of null

```

**Comment by jzohrab** [comment]
The front page (https://www.coronavirus.kdheks.gov/) has a link to a PDF doc, which we appear to have parsed in the past.  Link example:

```
<a href="/DocumentCenter/View/984/4-29-20-update-numbers" target="_blank" class="fancyButton fancyButton119 " aria-label="
		
			3,738 Confirmed Cases in Kansas (PDF)
		
	 Opens in new window" rel="noopener">
		<span>
			<span><span class="text">3,738 Confirmed Cases in Kansas (PDF)</span></span>
		</span>
	</a>
```

The underlying https://www.coronavirus.kdheks.gov/DocumentCenter/ doesn't appear easily scraped itself, so I'll get the file from the above.

**Comment by jzohrab** [comment]
Fixed in #917, waiting for merge.
